tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -84,12 +84,13 @@ Verify User Is Able to Activate Anaconda Commercial Edition
   Wait Until JupyterLab Code Cell Is Not Active
   Run Cell And Check For Errors  import numpy as np
   Capture Page Screenshot  conda_lib_install_result.png
+  Verify Library Version Is Greater Than  jupyterlab  3.1.4
+  Verify Library Version Is Greater Than  notebook    6.4.1
```

Comment:
You should only check for `x.y` as that is what we do in all other TCs and what we report in the documentation. The `z` version is bound to change frequently

Reply:
this TC is about checking the security issue on jupyter images whose solution was to have specific library versions. And the security fix has been implemented into .z version. https://blog.jupyter.org/cve-2021-32797-and-cve-2021-32798-remote-code-execution-in-jupyterlab-and-jupyter-notebook-a70fae0d3239